apps/demo/src/scss/styles.scss

```diff
@@ -5,8 +5,8 @@
 ************************************/
 
 // Imports
-@import 'daff-typography';
-@import 'daff-theme';
+@import 'daff-typography.scss';
```

Comment:
This change along with the changes in `theme.scss` and `utilities.scss` were to fix a bug on windows. Where the build could not differentiate between .css and .scss files.

Reply:
I don't believe there is a `daff-typography.css` file?

Reply:
@Nolan-Arnold and I resolved this offline and realised that he had a auto scss compiler installed.